srcpkgs/python3-PyQt5-webengine/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'python3-PyQt5-webengine'
+pkgname=python3-PyQt5-webengine
+version=5.15.0
+revision=1
+wrksrc="PyQtWebEngine-${version}"
+hostmakedepends="pkg-config qt5-qmake python3-PyQt5 sip5"
+makedepends="qt5-declarative-devel qt5-webchannel-devel qt5-location-devel
+ qt5-webengine-devel python3-devel python3-PyQt5-devel"
+depends="python3-PyQt5>=${version}_1 python3-PyQt5-webchannel>=${version}_1"
+short_desc="Python 3 bindings for the Qt5 toolkit - webengine module"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="GPL-3.0-only"
+homepage="https://www.riverbankcomputing.com/software/pyqtwebengine/intro"
+distfiles="${PYPI_SITE}/P/PyQtWebEngine/PyQtWebEngine-${version}.tar.gz"
+checksum=670812688e40bf75f70ddf01eadd897d231300318d3856b275bf8e7e0085bf75
+lib32disabled=yes
+nocross="configure script is broken for cross builds"
+
+case "$XBPS_TARGET_MACHINE" in
+	ppc64le*) ;;
+	ppc*) broken="no webengine for big endian ppc";;
+	arm*) broken="depends on qt5-webengine";;
+esac
+
+pre_build() {
+	rm -rf pyuic/uic/port_v2
+}
+
+do_build() {
+	python3 configure.py --no-dist-info \
```

Comment:
>`--no-dist-info` 

Due to this if a package has `PyQt5` specified as an requirement in `setup.py`, it will fail to find `PyQt5` even if it's installed. This flag should be removed I guess.

Reply:
That sounds reasonable.
Do you which package is that?
Or should we just enable it unconditionally :-p